### PR TITLE
Remove cleanup of experiments that are no longer running

### PR DESCRIPTION
**Pull request by christibbs**
We've made a product decision to not clean these up for now.

Impact: if a user stops an experiment via the console, all events will continue to be striped with the CUP for that experiment.

Also fixed the version number for the ABT dependency, which somehow got reverted. FIAM should update on minor ABT releases.

### CODE REVIEW

File: FirebaseInAppMessaging.podspec [hunk 1]
```diff
@@ -45,7 +45,7 @@ See more product details at https://firebase.google.com/products/in-app-messagin
   s.ios.dependency 'FirebaseAnalyticsInterop', '~> 1.3'
   s.dependency 'FirebaseInstanceID', '~> 4.0'
   s.dependency 'GoogleDataTransportCCTSupport', '~> 2.0'
-  s.dependency 'FirebaseABTesting', '~> 3.2'
+  s.dependency 'FirebaseABTesting', '~> 3.2.0'
```
Comment (paulb777) [inline FirebaseInAppMessaging.podspec]: Please revert. Firebase should only allow other pods in this repo to update minor versions.

Comment (morganchen12) [inline FirebaseInAppMessaging.podspec]: Is this change necessary?

Comment (christibbs) [reply FirebaseInAppMessaging.podspec]: Reverted.

**Comment by JasonAHeron** [comment]
Do you need to change any tests?

**Comment by christibbs** [comment]
> Do you need to change any tests?

Nope, apparently that was a hole in the unit test coverage.
